frontend/public/components/pod.jsx

```diff
@@ -279,7 +279,9 @@ const filters = [{
     { id: 'Pending', title: 'Pending' },
     { id: 'Terminating', title: 'Terminating' },
     { id: 'CrashLoopBackOff', title: 'CrashLoopBackOff' },
-    { id: 'Succeeded', title: 'Succeeded' },
+    // Use title "Completed" to match what appears in the status column for the pod.
+    // The pod phase is "Succeeded," but the container state is "Completed."
+    { id: 'Succeeded', title: 'Completed' },
```

Comment:
@TheRealJon I debated making `podPhaseFilterReducer` return "Completed" if that was the container container state, but I was worried we could have other "Succeeded" pods with some other container state. Then we'd be back to the problem where some pods didn't match a filter. I can't decide what's best. Let me know what you think.

Reply:
Yeah, I don't know if there are any other container states that would correspond to a pod 'Succeeded' phase, but I think this way works. Maybe we should consider showing pod phase in the table so that there is less confusion about the filter values? The filter values at the top are rarely going to match exactly what's in the status column, so we could end up with more bug reports about this kind of thing.